Makefile

```diff
@@ -72,12 +76,12 @@ unittests:
 	@echo "Running unit tests"
 	@tox -e unit -- ${UNIT_ARGS}
 
-functional:
-	@echo "Executing functional tests(charm will be build during functional tests)"
-	@tox -e func -- ${FUNC_ARGS}
+functional: 
```

Comment:
It seems that a `build` target is needed here.
```suggestion
functional: build
```

Reply:
No it's not. The pytest-operator will handle charm build for us.